src/modules/display.js

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line import/no-cycle
+import complete from './complete.js';
+// eslint-disable-next-line import/no-cycle
+import targeter from './targeter.js';
+
+const tasklist = document.querySelector('#dynamiccontainer');
+
+const display = () => {
+  tasklist.innerHTML = '';
+  let arr = JSON.parse(window.localStorage.getItem('tasklist'));
+  if (arr === null) {
+    arr = [];
+  }
+  arr.forEach((item) => {
+    if (item.completed === true) {
+      tasklist.innerHTML += `<div class="field" id=${item.index}>
+    <div class="inputcontainer">
+      <label for=${item.index}><input type="checkbox" class="task" checked><div class="tasktext ${item.completed}">${item.description}</div></label>
+    </div>
+    <button class="listbutton" id=button${item.index} type="button" title="Change Task"><ion-icon name="ellipsis-vertical"></ion-icon></button>
+  </div>`;
+    } else {
+      tasklist.innerHTML += `<div class="field" id=${item.index}>
+    <div class="inputcontainer">
+      <label for=${item.index}><input type="checkbox" class="task"><div class="tasktext ${item.completed}">${item.description}</div></label>
+    </div>
+    <button class="listbutton" id=button${item.index} type="button" title="Change Task"><ion-icon name="ellipsis-vertical"></ion-icon></button>
+  </div>`;
+    }
```

Comment:
In order to follow Javascript best practices and to simplify the code, you should use a `Ternary` here or a `Logical And` it would be like below:
 - ternary: `${item.completed}? <div>//HTML for completed case goes here</div>` : `<div>// HTML for not completed case goes here</div>`
 - logical and: `${item.completed} && <div>//HTML for completed case goes here</div>`
 
The only issue with the`logical and` is that you won't be able to add the incomplete case, read more about it using this [link](https://codeburst.io/javascript-short-circuit-conditionals-bbc13ac3e9eb)

Reply:
That's a very interesting alternative.